### DISCUSSION TITLE
Do you plan to add INTERSPEECH 2021 ?

### DISCUSSION THREAD

**Discussion by AlexeyBoiler** [asteroid-team/asteroid#481] [General]
Hello. Thank you for Asteroid. Microsoft has updated its library, everything in Asteroid DNSDataset is updated automatically?

**Comment by mpariente** [comment]
No, it isn't. 
And I haven't followed the changes. Would you like to update it?

**Reply by AlexeyBoiler** [reply]
Yes, I think about it. It's convenient to use your repository, but the new version DNSDataset may have better noise reduction quality.

**Reply by mpariente** [reply]
The dataset has better noise reduction?

**Reply by AlexeyBoiler** [reply]
I have not tested it, but I saw that it has been updated.
